Sources/StreamVideo/Utils/UnfairQueue/UnfairQueue.swift

```diff
@@ -23,15 +23,15 @@ import Foundation
 /// additional overhead of dispatching tasks and managing thread execution in a DispatchQueue could result
 /// in unnecessary latency, making `os_unfair_lock` the superior choice for scenarios where rapid, lightweight
 /// synchronization is paramount.
-final class UnfairQueue {
+public final class UnfairQueue {
```

Comment:
Hmm, seeing this one again - it was also in the other PR.

Reply:
As we discussed this is because both PRs need the file and i wanted to have on GH and succeed on tests. Once one of them get merged the file should disappear from the other PR